libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -3601,6 +3602,30 @@ void GCS_MAVLINK::send_matternet_FTS_version(void) const
     }
 }
 
+/*
+  send active rangefinders' version strings
+ */
+void GCS_MAVLINK::send_rangefinder_versions(void) const
+{
+
+    RangeFinder *rangefinder = RangeFinder::get_singleton();
+    if (rangefinder == nullptr) {
+        return;
+    }
+
```

Comment:
Create a local char array here to pass to get_version(). 

char buffer[80]

Reply:
Fixed!